source/content/guides/wordpress-composer/02-wordpress-composer-managed.md

```diff
@@ -0,0 +1,111 @@
+---
+title: WordPress with Composer on Pantheon
+subtitle: Create a Composer-managed WordPress Site with Bedrock
+description: Learn more about Bedrock and Composer-managed WordPress sites.
+categories: [develop]
+tags: [wordpress]
+contributors: [whitneymeredith, jspellman814, jazzsequence]
+layout: guide
+showtoc: true
+permalink: docs/guides/wordpress-composer/wordpress-composer-managed
+anchorid: wordpress-composer-managed
+---
+
+<Alert title="Early Access" type="info" icon="leaf">
+
+The WordPress Composer Managed upstream is available for Early Access participants. Features for WordPress Composer Managed are in active development. Pantheon's development team is rolling out new functionality often while this product is in Early Access. To learn how you can enroll in our Early Access program, visit [#wordpress in our community Slack](https://pantheon-community.slack.com/archives/CT8MC5Y0K).
+
+</Alert>
+
+This section provides information on how to use Bedrock with Integrated Composer on a WordPress site.
+
+WordPress does not natively support [Composer](https://getcomposer.org/), however, [Bedrock](https://roots.io/bedrock/) is a WordPress-specific framework for using Composer on WordPress sites.
+
+## Requirements
+
+- [PHP version](/php-versions#verify-current-php-versions) 7.4 or greater
```

Comment:
Are there still issues running WP 6.x and PHP 8.1?

Reply:
WP 6.x support for PHP 8.x is in "beta". I'm not sure that's relevant to call out here, I think it overcomplicates things.

Reply:
Here's WordPress' PHP version support chart: https://make.wordpress.org/core/handbook/references/php-compatibility-and-wordpress-versions/#supported-version-chart.

Don't think we need to call it out here, as it's mentioned on our own [Supported PHP Version page](https://pantheon.io/docs/supported-php), but I'd make that change if someone felt otherwise.